components/style/properties/declaration_block.rs

```diff
@@ -984,7 +984,13 @@ impl<'a, 'b, 'i> DeclarationParser<'i> for PropertyDeclarationParser<'a, 'b> {
         };
         input.parse_until_before(Delimiter::Bang, |input| {
             PropertyDeclaration::parse_into(self.declarations, id, self.context, input)
-                .map_err(|e| e.into())
+                .map_err(|e| {
+                    if let PropertyDeclarationParseError::InvalidValue(_, e) = e {
```

Comment:
Rather than rewriting the error in this particular spot (which misses a similar issue in parse_one_declaration_into), let's add a new name argument to PropertyDeclaration::parse_into instead and that in the error message.